compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_terminator_gen.bal

```diff
@@ -140,11 +144,14 @@ type TerminatorGenerator object {
                 self.mv.visitVarInsn(ISTORE, lhsLndex);
             } else if (bType is bir:BArrayType ||
                         bType is bir:BMapType ||
+                        bType is bir:BErrorType ||
                         bType is bir:BTypeAny ||
                         bType is bir:BTypeNil ||
                         bType is bir:BObjectType ||
                         bType is bir:BUnionType) {
                 self.mv.visitVarInsn(ASTORE, lhsLndex);
+            } else if (bType is bir:BErrorType) {
+                self.mv.visitVarInsn(ASTORE, lhsLndex);
```

Comment:
we don't need this line, do we?

btw, this should give a compile error, if not its a bug in type narrowing i believe.

Reply:
Yes. We do not need this line since line 147 covers. This seems to be added again when fixing some conflicts. Any we need line 147 to return error type if not you will get an exception when returning the error.

 JVM generation is not supported for type {reasonType:"string", detailType:{constraint:"any"}} {}
	at compiler_backend_jvm:0.0.0:genCallTerm(jvm_terminator_gen.bal:154)

Reply:
Fixed with last commit